src/coupling/balance.cc

```diff
@@ -18,6 +18,7 @@
 
 #include <iostream>
 #include <iomanip>
+#include <unordered_map>
```

Comment:
Already included in balance.hh.

Reply:
Never mind its better practice to include everything used in the actual source file.